core/test/utils/fixtures/data-generator.js

```diff
@@ -367,25 +372,41 @@ DataGenerator.Content = {
             event: 'subscriber.removed',
             target_url: 'https://example.com/webhooks/subscriber-removed'
         }
+    ],
+
+    integrations: [
+        {
+            id: ObjectId.generate(),
+            name: 'Test Integration',
+            slug: 'test-integration'
+        }
+    ],
+
+    api_keys: [
+        {
+            id: ObjectId.generate(),
+            type: 'admin'
+            // integration_id: DataGenerator.Content.integrations[0].id
```

Comment:
Is that on purpose?

Reply:
Yes, it was a comment to show what the integration_id will eventually be when it's [set up later on](https://github.com/TryGhost/Ghost/pull/9904/files/df2833c854ee13abf508c4db058746ada53ae841#diff-36ae953f72910a8d0e6a62f7dee1ca74R406)

Reply:
Ok 👍